docker-compose.yml

```diff
@@ -8,17 +8,22 @@ services:
     volumes:
       - "./files/:/etc/selenoid/" # assumed current dir contains browsers.json
       - "/var/run/docker.sock:/var/run/docker.sock"
+    container_name: selenoid
   selenoid-chrome:
     image: "selenoid/chrome:latest"
     network_mode: bridge
   selenoid-ui:
     image: "aerokube/selenoid-ui"
     network_mode: bridge
-    links:
-      - selenoid
     ports:
-      - "8080:8080"
+      - "8060:8080"
```

Comment:
tu nie powinno być 8060:8060 ?

Reply:
Dlaczego?
localhost:8060 zmapuje się do docker_conatiner:8080
celowo zmieniłem ten pierwszy, bo jednak sporo usług UI ma default 8080 i się potem gryzą np report-portal i selenoid
z kolei w kontenerze, jako ze sa w izolaci i maja wlasne adresy to juz kazdy bedzie mial swoje 8080 wolne :)